.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,29 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    strategy:
+      fail-fast: false
```

Comment:
I don't think fail-fast should be False. (One of the main) The point of linting is to avoid running the tests if there are major errors that can be found just by code inspection

Reply:
This is useful when testing across Linux, macOS, and Windows and/or multiple versions of Python (the comments that we deleted) as if all a Windows test fails, we can still see if macOS and Linux pass.  Give that we only have one run, fast-fail is a noop.

Reply:
I see, I thought it was for the case when running multiple jobs in parallel (e.g. linting and testing)